alias.ts

```diff
@@ -29,6 +29,8 @@ export const alias: Record<string, string> = {
   '@unocss/shared-common': r('./packages/shared-common/src/'),
   '@unocss/shared-docs': r('./packages/shared-docs/src/'),
   '@unocss/shared-integration': r('./packages/shared-integration/src/'),
+  '@unocss/svelte-preprocess': r('./packages/svelte-scoped/preprocessor/src/'),
+  '@unocss/svelte-scoped-vite': r('./packages/svelte-scoped/vite/src/'),
```

Comment:
Can we flat these nested packages, or we publish one package with sub modules, like `@unocss/svelte-scoped/preprocess` / `@unocss/svelte-scoped/vite`?

Reply:
https://github.com/unocss/unocss/blob/ab86ae7c7b1441ac1257fac8864a2f4f706f3c68/packages/preset-mini/package.json#L28-L37
https://github.com/unocss/unocss/blob/ab86ae7c7b1441ac1257fac8864a2f4f706f3c68/packages/preset-mini/build.config.ts#L4-L11

Here are the references

Reply:
Ok, done